statics/js/components/topology.js

```diff
@@ -234,6 +238,7 @@ var TopologyComponent = {
     });
 
     this.setFilterFromConfig();
+    this.topologyFilterQuery();
```

Comment:
this call not applying the filter, not sure from where we need to call this function. needs @safchain blessings :)

Reply:
@masco-enovance it does for me

Reply:
oh, cool. thanks sylvain!

Reply:
It does for me, what do you mean by not called, you never pass through "mounted" or the function is called but the filter is not applied ?

Reply:
yeah, I mean that the filter is not applied.